src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -110,10 +119,11 @@ export const MessagingThreadListItem: React.FC<
           {subContent && (
             <FlexBox alignItems={ItemAlign.CENTER}>
               <div className={subContentClass}>{subContent}</div>
-              {indicator}
+              {timestamp && indicator}
             </FlexBox>
           )}
         </FlexItem>
+        <FlexItem alignItems={ItemAlign.CENTER}>{subContent && !timestamp && indicator}</FlexItem>
```

Comment:
If I'm reading this right, this is rendering the indicator. But isn't that also being done on line 122?

Reply:
Commented above, but maybe this is too confusing at first glance... I'll think of how to make it more clear.